met/src/tools/other/pb2nc/pb2nc.cc

```diff
@@ -2920,39 +2920,65 @@ int combine_tqz_and_uv(map<float, float*> pqtzuv_map_tq,
       float *pqtzuv_tq, *pqtzuv_uv;
       float *pqtzuv_merged = (float *) 0;
       float *next_pqtzuv, *prev_pqtzuv;
+      float tq_pres_max, tq_pres_min, uv_pres_max, uv_pres_min;
       std::map<float,float*>::iterator it, it_tq, it_uv;
 
       // Gets pressure levels for TQZ records
-      for (it=pqtzuv_map_tq.begin(); it!=pqtzuv_map_tq.end(); ++it) {
-         tq_levels.add(int(it->first));
+      it = pqtzuv_map_tq.begin();
+      tq_pres_min = tq_pres_max = it->first;
+      for (; it!=pqtzuv_map_tq.end(); ++it) {
+         float pres_v = it->first;
+         if (tq_pres_min > pres_v) tq_pres_min = pres_v;
+         if (tq_pres_max < pres_v) tq_pres_max = pres_v;
+         tq_levels.add(nint(pres_v));
       }
       // Gets pressure levels for common records
-      for (it=pqtzuv_map_uv.begin(); it!=pqtzuv_map_uv.end(); ++it) {
-         if (tq_levels.has(int(it->first))) {
-            common_levels.add(int(it->first));
+      it = pqtzuv_map_uv.begin();
+      uv_pres_min = uv_pres_max = it->first;
+      for (; it!=pqtzuv_map_uv.end(); ++it) {
+         float pres_v = it->first;
+         if (uv_pres_min > pres_v) uv_pres_min = pres_v;
+         if (uv_pres_max < pres_v) uv_pres_max = pres_v;
+         if (tq_levels.has(nint(pres_v))) {
+            common_levels.add(nint(pres_v));
          }
       }
 
       if(mlog.verbosity_level() >= PBL_DEBUG_LEVEL) {
          log_tqz_and_uv(pqtzuv_map_tq, pqtzuv_map_uv, method_name);
       }
 
+      bool no_overlap = (tq_pres_max < uv_pres_min) || (tq_pres_min > uv_pres_max);
+      mlog << Debug(6) << method_name << "TQZ pressures: " << tq_pres_max
```

Comment:
```suggestion
      mlog << Debug(6) << "\n" << method_name << "TQZ pressures: " << tq_pres_max
```
Like Error messages, the Warning messages in MET start with a new line and end with a new line to make them stand out in the log file.